src/main.cpp

```diff
@@ -242,8 +242,6 @@ void unitTests()
 
 int mainEmulatorHLE()
 {
-	if (!TestWriteAccess(ActiveSettings::GetPath()))
-		wxMessageBox("Cemu doesn't have write access to it's own directory.\nPlease move it to a different location or run Cemu as administrator!", "Warning", wxOK|wxICON_ERROR); // todo - different error messages per OS
```

Comment:
Should this check be reinserted somewhere else?

Reply:
It tests write access in ActiveSettings::LoadOnce for all paths (except data_path, since it should only read from it), although it doesn't display a message but it writes in the log file.

Reply:
I think it would be a good idea to show the message box popup in addition to logging.

Reply:
Now it should also display a message box saying which folder failed the write access test.